<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DefringeForm</class>
 <widget class="QWidget" name="DefringeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>151</width>
    <height>117</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="MainLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ptInput" name="Radius"/>
   </item>
   <item>
    <widget class="ptInput" name="Threshold"/>
   </item>
   <item>
    <widget class="ptInput" name="MaskTuning"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="ptCheck" name="ColorCyan"/>
       </item>
       <item row="1" column="1">
        <widget class="ptCheck" name="ColorBlue"/>
       </item>
       <item row="0" column="1">
        <widget class="ptCheck" name="ColorYellow"/>
       </item>
       <item row="1" column="2">
        <widget class="ptCheck" name="ColorPurple"/>
       </item>
       <item row="0" column="2">
        <widget class="ptCheck" name="ColorGreen"/>
       </item>
       <item row="0" column="0">
        <widget class="ptCheck" name="ColorRed"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ptCheck</class>
   <extends>QCheckBox</extends>
   <header>ptCheck.h</header>
  </customwidget>
  <customwidget>
   <class>ptInput</class>
   <extends>QProgressBar</extends>
   <header>ptInput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
